### PR TITLE
 two sql Questions added

### PR DESCRIPTION
**Pull request by GulabSinghSikarwar**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,25 @@
 const sqlQuiz = [
+  {
+    Question: "In an RDBMS relationship between tables are created by using ",
+    Answer: "Foreign Keys",
+    Distractor1: "Candidate Keys ",
+    Distractor2: "Alternate Keys",
+    Distractor3: "Composite Keys",
+    Explanation:
+      "A foreign key is a column/group of columns used to make reference/link to another table's primary key.In other words, it is the representation of another table's primary key.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:9]:
```suggestion
      "A foreign key is used to link two tables by making reference to another table's primary key.",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+  {
+    Question:
+      "In order to add a new column to an exsisting table in SQL,we can use the command",
+    Answer: "ALTER TABLE",
+    Distractor1: "MODIFY TABLE",
+    Distractor2: "EDIT TABLE",
+    Distractor3: "ALTER COLUMNS",
+    Explanation:
+      "To add a new column, you first need to select the table with ALTER TABLE table_name, and then write the name of the new column and its datatype with ADD column_name datatype.",
+    Link: "https://www.freecodecamp.org/news/alter-table-in-sql-how-to-add-a-column-or-rename-a-column-in-sql/",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:21]: We already have a question for the ALTER TABLE statement.
Please remove this question.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/sql-quiz.ts (hunk 2 above)
Comment (GulabSinghSikarwar) [reply src/data/sql-quiz.ts:21]: Thanks, @jdwilkin4  for reviewing, I will surely make the requested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**